tests/integration/services.test.js

```diff
@@ -154,123 +155,108 @@ describe('GET /services', () => {
 
 describe('PUT /services/:id', () => {
   let authToken = null;
-  beforeEach(populateDB);
-  beforeEach(async () => {
-    authToken = await getToken('client_admin');
-  });
-  it('should update service', async () => {
-    const payload = { defaultUnitAmount: 15, name: 'Service bis', startDate: '2019-01-16 17:58:15.519', vat: 12 };
-    const response = await app.inject({
-      method: 'PUT',
-      url: `/services/${servicesList[0]._id.toHexString()}`,
-      headers: { Cookie: `alenvi_token=${authToken}` },
-      payload,
+  describe('CLIENT_ADMIN', () => {
+    beforeEach(populateDB);
+    beforeEach(async () => {
+      authToken = await getToken('client_admin');
     });
+    it('should update service', async () => {
+      const payload = { defaultUnitAmount: 15, name: 'Service bis', startDate: '2019-01-16 17:58:15.519', vat: 12 };
+      const response = await app.inject({
+        method: 'PUT',
+        url: `/services/${servicesList[0]._id.toHexString()}`,
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload,
+      });
 
-    expect(response.statusCode).toBe(200);
-
-    const updatedService = await Service.findOne({ _id: servicesList[0]._id.toHexString() }).lean();
-    expect(updatedService.versions).toBeDefined();
-    expect(updatedService.versions.length).toBe(servicesList[0].versions.length + 1);
-  });
-
-  it('should archive service', async () => {
-    const payload = { isArchived: true };
-    const response = await app.inject({
-      method: 'PUT',
-      url: `/services/${servicesList[0]._id.toHexString()}`,
-      headers: { Cookie: `alenvi_token=${authToken}` },
-      payload,
+      expect(response.statusCode).toBe(200);
+      const updatedService = await Service.findOne({ _id: servicesList[0]._id }).lean();
+      expect(updatedService.versions.length).toBe(servicesList[0].versions.length + 1);
     });
 
```

Comment:
Sur le test en dessous de cette ligne (`should archive service`), il n'y a pas besoin de declarer le payload, tu peux directement l'utiliser dans l'appel a la route.

Reply:
done